### DISCUSSION TITLE
Code block text is "fuzzy" and no code highlight in visual editor

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#5276] [Error / Bug Report]
Hi there,

The text in code blocks seems to have a styling problem, it's "fuzzy"/has a backlight and is hard to read, but it changes a bit after selecting the text.

Image before:
![before](https://user-images.githubusercontent.com/5992169/167621306-692b730f-5c4c-4d5a-994f-4288af5f84f7.png)

Image after selecting everything with Ctrl + A:
![after](https://user-images.githubusercontent.com/5992169/167621398-9f44d81f-fa64-47d2-87ba-fda7679261f7.png)

**Comment by tmart-ops** [comment]
I see the fuzzy text as well, however when I select text it becomes unfuzzy. [Version 2.5.282]
Also wish we could modify font size in code blocks but that's a separate issue.

**Reply by danieldll099** [reply]
> I see the fuzzy text as well, however when I select text it becomes unfuzzy. [Version 2.5.282]
> 
> Also wish we could modify font size in code blocks but that's a separate issue.

**Comment by danieldll099** [comment]
Ok
